components/mpas-ocean/src/ocean.cmake

```diff
@@ -115,6 +115,7 @@ list(APPEND RAW_SOURCES
   core_ocean/shared/mpas_ocn_manufactured_solution.F
   core_ocean/shared/mpas_ocn_subgrid.F
   core_ocean/shared/mpas_ocn_scaled_dismf.F
+  core_ocean/shared/mpas_ocn_scaled_sfwf.F
```

Comment:
This probably needs to be added to the standalone Makefile as well?   Have you verified that standalone still builds?  If not, one of the reviewers should do that.

Reply:
I take this back.  We don't need this feature in standalone builds.  So this is fine.

Reply:
@zhangshixuan1987 Am I correct in understanding that in standalone ocean we wouldn't ever call this module? I think maybe we don't want to add it to the standalone makefile. If you agree, I can still test to make sure standalone builds and runs as-is with this branch.